### PR TITLE
Add Withers thesis to publications

### PR DESCRIPTION
**Pull request by ljhwang**
First publication of 2021!

Adds Craig Withers master's thesis to the publication list.

### CODE REVIEW

File: doc/manual/citing_aspect.bib [hunk 1]
```diff
@@ -831,7 +831,18 @@ @ARTICLE{10.3389/feart.2020.533392
...
 }
+
+@mastersthesis{Withers2021,
+author="Withers, Craig",
+title="Modelling slab age and crustal thickness: numerical approaches to drivers of compressive stresses in the overriding plate in Andean style subduction zone systems",
+year="2021",
+publisher="Durham University",
+optlocation="Lorraine Hwang (ljhwang@ucdavis.edu)",
```
Comment (tjhei) [inline doc/manual/citing_aspect.bib:844]: would you mind removing this line?

Comment (ljhwang) [reply doc/manual/citing_aspect.bib]: Whoops.
Fixed a couple of artifacts from the export process. Thanks for the catch.

**Comment by tjhei** [comment]
Your last commit does not have an email address or your name  associated with it. Do you want to try to fix that?
See:
```
commit ecccd126e018123827100fc38343c99106ddb615 (HEAD -> pr3955)
Author: ljhwang <>
Date:   Thu Jan 21 11:01:11 2021 -0800

    delete unrelevant metadata from citation

commit 5797d2ad7465795166f289f741bac467eb2fbb01
Author: Lorraine Hwang <ljhwang@ucdavis.edu>
Date:   Wed Jan 20 16:27:47 2021 -0800

    Add Withers thesis to publications
```
( https://github.com/dealii/dealii/wiki/Indentation#commit-authorship )

**Comment by bangerth** [comment]
Or just git fixup the last commit into the first.

**Comment by bangerth** [comment]
Ping @ljhwang ?

**Comment by ljhwang** [comment]
Hmmm .... This may have something to do with Big Sur deleting my SSH key. Thought I had a quick work around.  Stay tuned.

**Comment by ljhwang** [comment]
Ok. Applied multiple fixes and added a new reference.  Let me know if this did not work or something else looks funny.

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
Thanks!

**Pull request merged**